ui/src/TextUpload.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import axios from 'axios';
+import './TextUpload.css'
 
-type TextProps = {}
+type TextProps = {
+    sendTopText: CallableFunction,
+    sendBottomText: CallableFunction
+}
 
 type TextState = {
     topText: String,
     bottomText: String,

```

Comment:
Do we need to maintain these in this state anymore?  We've moved the state to the parent component and use prop functions to pass the data up.  These aren't used in this component anywhere.

Reply:
I am using the topText and bottomText state when I send it off in the post request. Is this not necessary?

Reply:
Oh, that makes sense, the `POST` does get called from this component, doesn't it?